File: examples_py/c2_switch.py
print("""
What is your favortie color (input the number):
      1. Red
      2. Blue
      3. Green
      4. Yellow
""")

user_input = int(input())

if user_input == 1:
    print("You must like apples!\n")
elif user_input == 2:
    print("You must like looking at the sky!\n")
elif user_input == 3:
    print("You must like vegetables!\n")
elif user_input == 4:
    print("You must like looking at the Sun! (And you should probably stop...)\n")
else:
    print("I've never heard of that color...:(\n")

File: examples_cpp/c2_switch.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    int input;
    cout << "What is your favorite color (input the number):\n";
    cout << "1. Red \n2. Blue\n3. Green\n4. Yellow\n";
    cin >> input;

    switch (input){
        case 1:
            cout << "You must like apples!\n";
            break;
        case 2:
            cout << "You must like looking at the sky!\n";
            break;
        case 3:
            cout << "You must like vegetables!\n";
            break;
        case 4:
            cout << "You must like looking at the Sun! (And you should probably stop...)\n";
            break;
        default:
            cout << "I've never heard of that color... :(\n";
    }
    return 0;
}